airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/DebeziumRecordPublisher.java

```diff
@@ -85,6 +89,7 @@ public void start(Queue<ChangeEvent<String, String>> queue) {
         .using((success, message, error) -> {
```

Comment:
this callback only gets invoked when the engine is totally off i'm assuming?

Reply:
yep, it's the last thing